# request
string tip_name
float32[] geometry
float32[] current_joint_angles

---
# response
float32[] target_joint_angles
